src/components/implementation/tests/unit_cxx/init.c

```diff
@@ -0,0 +1,12 @@
+#include <cos_component.h>
+#include <cobj_format.h>
+#include <cos_kernel_api.h>
+#include "init.h"
+
+struct cos_compinfo parent_cinfo;
+
+void init_cinfo(void){
```

Comment:
This is interesting.  I suppose we really can have a default initialization for all of this.  This should likely be in the `cos_defkernel_api` so that we can use it elsewhere.  We *might* already have it there!  Please check.  If not, we'll want two functions.  One for the "current" component's compinfo, and one for a "child" components.  Either feel free to add this, or add an issue in the issue tracker to "Create default cos_compinfo initialization functions".